WooCommerce/Classes/Model/ShippingLabelPackageInfo.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+/// A structure to hold the basic details for a selected package on Package Details screen of Shipping Label purchase flow.
+///
+struct ShippingLabelPackageInfo: Hashable, Equatable {
```

Comment:
I think this name works, but just thinking about possible alternate names:

What do you think of naming this `ShippingLabelPackageDetails`? Or would that get confusing because that's also the name of the Package Details screen? Another idea that comes to mind is `ShippingLabelPackageAttributes`.

Reply:
Thanks - I like the name `ShippingLabelPackageAttributes`! I don't think we can use `ShippingLabelPackageDetails` as we can't have two files of the same name in the same module.